corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -298,6 +298,13 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         caseListSearch: function (e) {
             e.preventDefault();
             var searchText = $('#searchText').val();
+            if (hqImport('hqwebapp/js/toggles').toggleEnabled('APP_ANALYTICS')) {
+                hqImport('analytix/js/kissmetrix').track.event('Searched Case List', {
+                    domain: FormplayerFrontend.getChannel().request("currentUser").domain,
+                    app_id: FormplayerFrontend.getChannel().request('getCurrentAppId'),
+                    name: searchText,
```

Comment:
This shouldn't include the search text itself, which might be sensitive data. It should include the name of the menu being searched.

Reply:
Thinking about this some more: viewing a case list and searching a case list are the same request. The only difference is whether or not the search text is present. So you'll likely be recording the view and search events in the same place.